scripts/create_gitlab_snapshot.py

```diff
@@ -87,16 +112,16 @@ def create_snapshot(volume: Mapping[str, JSON]):
                                            dict(ResourceType='snapshot',
                                                 Tags=[{'Key': k, 'Value': v} for k, v in tags.items()])
                                        ])
-    snapshot = [response['SnapshotId']]
-    log.info('Created snapshot %s for %s, waiting for completion',
+    snapshot = response['SnapshotId']
+    log.info('Completing snapshot %r for %r …',
```

Comment:
This line is in the wrong commit. I think there are more like this.

```suggestion
    log.info('Snapshot %r of volume %r is being created …',
```